src/routes/request.js

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+import requestController from '../controllers/requestController';
+import { requestSchema } from '../validation/schemas';
+import validate from '../validation/validator';
+import verifyUser from '../middlewares/auth';
+
+const router = Router();
+
+router.patch('/request/:id', validate(requestSchema, 'body'), verifyUser, requestController.updateRequest);
```

Comment:
Since the root route will be `requests`, this route should be `/:id` not `/request/:id`

Also try and apply destructuring in your code. 

Example
Instead of having `requestController.updateRequest`

you can destructure it up by:
```
const { update } = requestController;

router.patch(':id', validate(requestSchema, 'body'), verifyUser, update);

```

You might not necessarily need to add `Request` to the name of the functions in the request controller.